routes/index.js

```diff
@@ -11,6 +11,61 @@ var officegen = require('officegen');
 var nodemailer = require('nodemailer')
 var config = require('config')
 
+router.get('/updateschema', function(req, res, next) {
```

Comment:
It might be better not to put this on a route...or if you want to, check credentials and only allow admin to do it.